### DISCUSSION TITLE
GPU process isn't usable. Goodbye.

### DISCUSSION THREAD

**Discussion by christophehenry88** [cypress-io/cypress#18818] [Questions and Help]
Hello,

I installed Cypress on my windows (via npm) as explained on https://docs.cypress.io/guides/getting-started/installing-cypress

When i run the command "cypress run" , i have the error :  GPU process isn't usable. Goodbye.

The full log is in attachment.

[full_log.txt](https://github.com/cypress-io/cypress/files/7497323/full_log.txt)

**Comment by deimonbody** [comment]
Have you resolved the problem ? I currently stacked at that :(
